lesson18/src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Yomogi&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/reset.css" />
+  <link rel="stylesheet" href="./css/styles.css" />
+  <script type="module" src="./js/main.js" defer></script>
+</head>
```

Comment:
IMO 
> The defer attribute has no effect on [module scripts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) — they defer by default.

[ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attributes)

Reply:
thank you!!
For the first time I knew that if I used type = module, it would be loaded last.

I'm not using module this time, 
so Removed the type attribute.

[7d05ada](https://github.com/nyattsu72/hanson-frontend-cource/pull/26/commits/7d05ada7985f4a3da1fd1f53c89be16974a71c7d)